pkg/core/storage/store.go

```diff
@@ -39,6 +39,9 @@ const (
 	// unfinished state reset and to 0 on unfinished state jump).
 	SYSStateChangeStage KeyPrefix = 0xc4
 	SYSVersion          KeyPrefix = 0xf0
+	// SYSStateSyncCheckpoint is used to store checkpoint of state synchronization
+	// process performed by `statesync.Module`.
+	SYSStateSyncCheckpoint KeyPrefix = 0x05
```

Comment:
It still uses 0x05, we can't take this byte. Use next available prefix from SYS-range, it's 0xc5.